.rubocop.yml

```diff
@@ -53,27 +53,6 @@ Naming/VariableNumber:
     - 'lib/facter/resolvers/windows/ffi/networking_ffi.rb'
     - 'lib/facter/util/facts/windows_release_finder.rb'
 
-RSpec/MultipleMemoizedHelpers:
```

Comment:
Was this, `RSpec/MultipleMemoizedHelpers` meant to be removed? I'm not tracking if the removal is directly related to the main issue of this PR.